src/main/java/com/epam/lab/paymentsystem/controller/OperationController.java

```diff
@@ -86,9 +87,9 @@ public String paymentOperation(@RequestParam(name = "src") Long srcId,
     } catch (UnsupportedOperationException e) {
       model.addAttribute("message", e.getMessage());
       LOGGER.error("Exception payment operation", e);
-      return operationPage;
+      return OPERATION_PAGE;
     }
-    return operationPage;
+    return REDIRECT_TO + "/" + userService.getCurrentUserLogin();
```

Comment:
As I see, you Autowire userService in order to redirect to user page and that's it. I dont really think it is worth it.

Reply:
I replaced it with retrieving same page with successful message